internal/cmd/local/command_kafka_start.go

```diff
@@ -26,6 +25,7 @@ import (
 
 var (
 	statusBlacklist = []string{"Pulling fs layer", "Waiting", "Downloading", "Download complete", "Verifying Checksum", "Extracting", "Pull complete"}
+	localArch       = "amd64"
```

Comment:
this is the default value. but using goreleaser file to build cli will potentially overwrite this